### PR TITLE
[Compute] Update Compute management client library to 20.0.0

### PR DESCRIPTION
**Pull request by hyonholee**
https://github.com/Azure/azure-powershell/issues/5956

https://github.com/JeffBow/AzurePowerShell/issues/15

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by cormacpayne** [comment]
@hyonholee Hey Hyonho, would you mind taking a look at the build errors after we merged in the latest from the `preview` branch?

**Comment by hyonholee** [comment]
@cormacpayne @praries880 I resolved merge conflict and updated the PR to address the comment.

File: src/ResourceManager/Compute/ChangeLog.md [hunk 1]
```diff
@@ -18,6 +18,10 @@
         - Additional information about change #1
 -->
 ## Current Release
+* Update Compute client library version to fix following cmdlets
+    - Grant-AzureRmDisk
```
Comment (praries880) [inline src/ResourceManager/Compute/ChangeLog.md]: Is this Grant-AzureRMDiskAccess?

File: src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachine/VirtualMachineRestartMethod.cs [hunk 2]
```diff
@@ -79,8 +79,7 @@ protected void ExecuteVirtualMachineRestartMethod(object[] invokeMethodInputPara
             string resourceGroupName = (string)ParseParameter(invokeMethodInputParameters[0]);
             string vmName = (string)ParseParameter(invokeMethodInputParameters[1]);
 
-            var result = VirtualMachinesClient.Restart(resourceGroupName, vmName);
-            WriteObject(result);
+            VirtualMachinesClient.Restart(resourceGroupName, vmName);
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachine/VirtualMachineRestartMethod.cs]: This seems like a breaking change, is this value written elsewhere?

File: src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachine/VirtualMachineRedeployMethod.cs [hunk 3]
```diff
@@ -79,8 +79,7 @@ protected void ExecuteVirtualMachineRedeployMethod(object[] invokeMethodInputPar
             string resourceGroupName = (string)ParseParameter(invokeMethodInputParameters[0]);
             string vmName = (string)ParseParameter(invokeMethodInputParameters[1]);
 
-            var result = VirtualMachinesClient.Redeploy(resourceGroupName, vmName);
-            WriteObject(result);
+            VirtualMachinesClient.Redeploy(resourceGroupName, vmName);
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachine/VirtualMachineRedeployMethod.cs]: This seems like a breaking change, is this written elsewhere?

File: src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachine/VirtualMachinePowerOffMethod.cs [hunk 4]
```diff
@@ -79,8 +79,7 @@ protected void ExecuteVirtualMachinePowerOffMethod(object[] invokeMethodInputPar
             string resourceGroupName = (string)ParseParameter(invokeMethodInputParameters[0]);
             string vmName = (string)ParseParameter(invokeMethodInputParameters[1]);
 
-            var result = VirtualMachinesClient.PowerOff(resourceGroupName, vmName);
-            WriteObject(result);
+            VirtualMachinesClient.PowerOff(resourceGroupName, vmName);
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachine/VirtualMachinePowerOffMethod.cs]: breaking change

File: src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachine/VirtualMachinePerformMaintenanceMethod.cs [hunk 5]
```diff
@@ -79,8 +79,7 @@ protected void ExecuteVirtualMachinePerformMaintenanceMethod(object[] invokeMeth
             string resourceGroupName = (string)ParseParameter(invokeMethodInputParameters[0]);
             string vmName = (string)ParseParameter(invokeMethodInputParameters[1]);
 
-            var result = VirtualMachinesClient.PerformMaintenance(resourceGroupName, vmName);
-            WriteObject(result);
+            VirtualMachinesClient.PerformMaintenance(resourceGroupName, vmName);
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachine/VirtualMachinePerformMaintenanceMethod.cs]: breaking change?

File: src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachine/VirtualMachineGeneralizeMethod.cs [hunk 6]
```diff
@@ -79,8 +79,7 @@ protected void ExecuteVirtualMachineGeneralizeMethod(object[] invokeMethodInputP
             string resourceGroupName = (string)ParseParameter(invokeMethodInputParameters[0]);
             string vmName = (string)ParseParameter(invokeMethodInputParameters[1]);
 
-            var result = VirtualMachinesClient.Generalize(resourceGroupName, vmName);
-            WriteObject(result);
+            VirtualMachinesClient.Generalize(resourceGroupName, vmName);
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachine/VirtualMachineGeneralizeMethod.cs]: breaking change?

File: src/ResourceManager/Compute/Commands.Compute/Generated/Snapshot/SnapshotRevokeAccessMethod.cs [hunk 7]
```diff
@@ -79,8 +79,7 @@ protected void ExecuteSnapshotRevokeAccessMethod(object[] invokeMethodInputParam
             string resourceGroupName = (string)ParseParameter(invokeMethodInputParameters[0]);
             string snapshotName = (string)ParseParameter(invokeMethodInputParameters[1]);
 
-            var result = SnapshotsClient.RevokeAccess(resourceGroupName, snapshotName);
-            WriteObject(result);
+            SnapshotsClient.RevokeAccess(resourceGroupName, snapshotName);
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/Snapshot/SnapshotRevokeAccessMethod.cs]: breaking change?

File: src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachine/VirtualMachineConvertToManagedDisksMethod.cs [hunk 8]
```diff
@@ -79,8 +79,7 @@ protected void ExecuteVirtualMachineConvertToManagedDisksMethod(object[] invokeM
             string resourceGroupName = (string)ParseParameter(invokeMethodInputParameters[0]);
             string vmName = (string)ParseParameter(invokeMethodInputParameters[1]);
 
-            var result = VirtualMachinesClient.ConvertToManagedDisks(resourceGroupName, vmName);
-            WriteObject(result);
+            VirtualMachinesClient.ConvertToManagedDisks(resourceGroupName, vmName);
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachine/VirtualMachineConvertToManagedDisksMethod.cs]: breaking change?

File: src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachine/VirtualMachineDeallocateMethod.cs [hunk 9]
```diff
@@ -79,8 +79,7 @@ protected void ExecuteVirtualMachineDeallocateMethod(object[] invokeMethodInputP
             string resourceGroupName = (string)ParseParameter(invokeMethodInputParameters[0]);
             string vmName = (string)ParseParameter(invokeMethodInputParameters[1]);
 
-            var result = VirtualMachinesClient.Deallocate(resourceGroupName, vmName);
-            WriteObject(result);
+            VirtualMachinesClient.Deallocate(resourceGroupName, vmName);
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachine/VirtualMachineDeallocateMethod.cs]: breaking change?

File: src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachine/VirtualMachineDeleteMethod.cs [hunk 10]
```diff
@@ -79,8 +79,7 @@ protected void ExecuteVirtualMachineDeleteMethod(object[] invokeMethodInputParam
             string resourceGroupName = (string)ParseParameter(invokeMethodInputParameters[0]);
             string vmName = (string)ParseParameter(invokeMethodInputParameters[1]);
 
-            var result = VirtualMachinesClient.Delete(resourceGroupName, vmName);
-            WriteObject(result);
+            VirtualMachinesClient.Delete(resourceGroupName, vmName);
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachine/VirtualMachineDeleteMethod.cs]: breaking change?

File: src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachine/VirtualMachineStartMethod.cs [hunk 11]
```diff
@@ -79,8 +79,7 @@ protected void ExecuteVirtualMachineStartMethod(object[] invokeMethodInputParame
             string resourceGroupName = (string)ParseParameter(invokeMethodInputParameters[0]);
             string vmName = (string)ParseParameter(invokeMethodInputParameters[1]);
 
-            var result = VirtualMachinesClient.Start(resourceGroupName, vmName);
-            WriteObject(result);
+            VirtualMachinesClient.Start(resourceGroupName, vmName);
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachine/VirtualMachineStartMethod.cs]: breaking change?

File: src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetDeallocateMethod.cs [hunk 12]
```diff
@@ -97,8 +97,7 @@ protected void ExecuteVirtualMachineScaleSetDeallocateMethod(object[] invokeMeth
                 instanceIds = inputArray2.ToList();
             }
 
-            var result = VirtualMachineScaleSetsClient.Deallocate(resourceGroupName, vmScaleSetName, instanceIds);
-            WriteObject(result);
+            VirtualMachineScaleSetsClient.Deallocate(resourceGroupName, vmScaleSetName, instanceIds);
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetDeallocateMethod.cs]: breaking change?

File: src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetDeleteMethod.cs [hunk 13]
```diff
@@ -79,8 +79,7 @@ protected void ExecuteVirtualMachineScaleSetDeleteMethod(object[] invokeMethodIn
             string resourceGroupName = (string)ParseParameter(invokeMethodInputParameters[0]);
             string vmScaleSetName = (string)ParseParameter(invokeMethodInputParameters[1]);
 
-            var result = VirtualMachineScaleSetsClient.Delete(resourceGroupName, vmScaleSetName);
-            WriteObject(result);
+            VirtualMachineScaleSetsClient.Delete(resourceGroupName, vmScaleSetName);
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetDeleteMethod.cs]: breaking change?

File: src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetPerformMaintenanceMethod.cs [hunk 14]
```diff
@@ -97,8 +97,7 @@ protected void ExecuteVirtualMachineScaleSetPerformMaintenanceMethod(object[] in
                 instanceIds = inputArray2.ToList();
             }
 
-            var result = VirtualMachineScaleSetsClient.PerformMaintenance(resourceGroupName, vmScaleSetName, instanceIds);
-            WriteObject(result);
+            VirtualMachineScaleSetsClient.PerformMaintenance(resourceGroupName, vmScaleSetName, instanceIds);
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetPerformMaintenanceMethod.cs]: breaking change?

File: src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetPowerOffMethod.cs [hunk 15]
```diff
@@ -97,8 +97,7 @@ protected void ExecuteVirtualMachineScaleSetPowerOffMethod(object[] invokeMethod
                 instanceIds = inputArray2.ToList();
             }
 
-            var result = VirtualMachineScaleSetsClient.PowerOff(resourceGroupName, vmScaleSetName, instanceIds);
-            WriteObject(result);
+            VirtualMachineScaleSetsClient.PowerOff(resourceGroupName, vmScaleSetName, instanceIds);
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetPowerOffMethod.cs]: breaking change?

File: src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetRedeployMethod.cs [hunk 16]
```diff
@@ -97,8 +97,7 @@ protected void ExecuteVirtualMachineScaleSetRedeployMethod(object[] invokeMethod
                 instanceIds = inputArray2.ToList();
             }
 
-            var result = VirtualMachineScaleSetsClient.Redeploy(resourceGroupName, vmScaleSetName, instanceIds);
-            WriteObject(result);
+            VirtualMachineScaleSetsClient.Redeploy(resourceGroupName, vmScaleSetName, instanceIds);
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetRedeployMethod.cs]: breaking change?

File: src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetReimageAllMethod.cs [hunk 17]
```diff
@@ -97,8 +97,7 @@ protected void ExecuteVirtualMachineScaleSetReimageAllMethod(object[] invokeMeth
                 instanceIds = inputArray2.ToList();
             }
 
-            var result = VirtualMachineScaleSetsClient.ReimageAll(resourceGroupName, vmScaleSetName, instanceIds);
-            WriteObject(result);
+            VirtualMachineScaleSetsClient.ReimageAll(resourceGroupName, vmScaleSetName, instanceIds);
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetReimageAllMethod.cs]: breaking change?

File: src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetReimageMethod.cs [hunk 18]
```diff
@@ -97,8 +97,7 @@ protected void ExecuteVirtualMachineScaleSetReimageMethod(object[] invokeMethodI
                 instanceIds = inputArray2.ToList();
             }
 
-            var result = VirtualMachineScaleSetsClient.Reimage(resourceGroupName, vmScaleSetName, instanceIds);
-            WriteObject(result);
+            VirtualMachineScaleSetsClient.Reimage(resourceGroupName, vmScaleSetName, instanceIds);
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetReimageMethod.cs]: breaking change?

File: src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetRestartMethod.cs [hunk 19]
```diff
@@ -97,8 +97,7 @@ protected void ExecuteVirtualMachineScaleSetRestartMethod(object[] invokeMethodI
                 instanceIds = inputArray2.ToList();
             }
 
-            var result = VirtualMachineScaleSetsClient.Restart(resourceGroupName, vmScaleSetName, instanceIds);
-            WriteObject(result);
+            VirtualMachineScaleSetsClient.Restart(resourceGroupName, vmScaleSetName, instanceIds);
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetRestartMethod.cs]: breaking change?

File: src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetStartMethod.cs [hunk 20]
```diff
@@ -97,8 +97,7 @@ protected void ExecuteVirtualMachineScaleSetStartMethod(object[] invokeMethodInp
                 instanceIds = inputArray2.ToList();
             }
 
-            var result = VirtualMachineScaleSetsClient.Start(resourceGroupName, vmScaleSetName, instanceIds);
-            WriteObject(result);
+            VirtualMachineScaleSetsClient.Start(resourceGroupName, vmScaleSetName, instanceIds);
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetStartMethod.cs]: breaking change?

File: src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetUpdateInstancesMethod.cs [hunk 21]
```diff
@@ -97,8 +97,7 @@ protected void ExecuteVirtualMachineScaleSetUpdateInstancesMethod(object[] invok
                 instanceIds = inputArray2.ToList();
             }
 
-            var result = VirtualMachineScaleSetsClient.UpdateInstances(resourceGroupName, vmScaleSetName, instanceIds);
-            WriteObject(result);
+            VirtualMachineScaleSetsClient.UpdateInstances(resourceGroupName, vmScaleSetName, instanceIds);
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetUpdateInstancesMethod.cs]: breaking change?

File: src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSetRollingUpgrade/VirtualMachineScaleSetRollingUpgradeCancelMethod.cs [hunk 22]
```diff
@@ -79,8 +79,7 @@ protected void ExecuteVirtualMachineScaleSetRollingUpgradeCancelMethod(object[]
             string resourceGroupName = (string)ParseParameter(invokeMethodInputParameters[0]);
             string vmScaleSetName = (string)ParseParameter(invokeMethodInputParameters[1]);
 
-            var result = VirtualMachineScaleSetRollingUpgradesClient.Cancel(resourceGroupName, vmScaleSetName);
-            WriteObject(result);
+            VirtualMachineScaleSetRollingUpgradesClient.Cancel(resourceGroupName, vmScaleSetName);
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSetRollingUpgrade/VirtualMachineScaleSetRollingUpgradeCancelMethod.cs]: breaking change?

File: src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSetRollingUpgrade/VirtualMachineScaleSetRollingUpgradeStartOSUpgradeMethod.cs [hunk 23]
```diff
@@ -79,8 +79,7 @@ protected void ExecuteVirtualMachineScaleSetRollingUpgradeStartOSUpgradeMethod(o
             string resourceGroupName = (string)ParseParameter(invokeMethodInputParameters[0]);
             string vmScaleSetName = (string)ParseParameter(invokeMethodInputParameters[1]);
 
-            var result = VirtualMachineScaleSetRollingUpgradesClient.StartOSUpgrade(resourceGroupName, vmScaleSetName);
-            WriteObject(result);
+            VirtualMachineScaleSetRollingUpgradesClient.StartOSUpgrade(resourceGroupName, vmScaleSetName);
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSetRollingUpgrade/VirtualMachineScaleSetRollingUpgradeStartOSUpgradeMethod.cs]: breaking change?

File: src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/AEMExtensionTests.cs [hunk 24]
```diff
@@ -24,8 +24,12 @@ public AEMExtensionTests(Xunit.Abstractions.ITestOutputHelper output)
             ServiceManagemenet.Common.Models.XunitTracingInterceptor.AddToContext(new ServiceManagemenet.Common.Models.XunitTracingInterceptor(output));
         }
 
-        [Fact(Skip = "Need service team to re-record test after changes to the ClientRuntime.")]
-        [Trait("Re-record", "ClientRuntime changes")]
+#if NETSTANDARD
+        [Fact(Skip = "Get-Location in Common.ps1 is not working correctly for NETSTANDARD")]
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/AEMExtensionTests.cs]: Please file an issue for the precise behavior you are seeing - we should not add tests in desktop without also adding them in netcore.

File: src/ResourceManager/Compute/Commands.Compute/Generated/AvailabilitySet/AvailabilitySetDeleteMethod.cs [hunk 25]
```diff
@@ -79,8 +79,7 @@ protected void ExecuteAvailabilitySetDeleteMethod(object[] invokeMethodInputPara
             string resourceGroupName = (string)ParseParameter(invokeMethodInputParameters[0]);
             string availabilitySetName = (string)ParseParameter(invokeMethodInputParameters[1]);
 
-            var result = AvailabilitySetsClient.Delete(resourceGroupName, availabilitySetName);
-            WriteObject(result);
+            AvailabilitySetsClient.Delete(resourceGroupName, availabilitySetName);
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/AvailabilitySet/AvailabilitySetDeleteMethod.cs]: breaking change?

File: src/ResourceManager/Compute/Commands.Compute/Generated/AvailabilitySet/AvailabilitySetUpdateMethod.cs [hunk 26]
```diff
@@ -0,0 +1,113 @@
+//
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/AvailabilitySet/AvailabilitySetUpdateMethod.cs]: Is this a new cmdlet?

File: src/ResourceManager/Compute/Commands.Compute/Generated/Disk/DiskDeleteMethod.cs [hunk 27]
```diff
@@ -79,8 +79,7 @@ protected void ExecuteDiskDeleteMethod(object[] invokeMethodInputParameters)
             string resourceGroupName = (string)ParseParameter(invokeMethodInputParameters[0]);
             string diskName = (string)ParseParameter(invokeMethodInputParameters[1]);
 
-            var result = DisksClient.Delete(resourceGroupName, diskName);
-            WriteObject(result);
+            DisksClient.Delete(resourceGroupName, diskName);
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/Disk/DiskDeleteMethod.cs]: what is this actually doing, with the method below calling async methods?  Which is actually used in cmdlet execution?

File: src/ResourceManager/Compute/Commands.Compute/Generated/Disk/DiskRevokeAccessMethod.cs [hunk 28]
```diff
@@ -79,8 +79,7 @@ protected void ExecuteDiskRevokeAccessMethod(object[] invokeMethodInputParameter
             string resourceGroupName = (string)ParseParameter(invokeMethodInputParameters[0]);
             string diskName = (string)ParseParameter(invokeMethodInputParameters[1]);
 
-            var result = DisksClient.RevokeAccess(resourceGroupName, diskName);
-            WriteObject(result);
+            DisksClient.RevokeAccess(resourceGroupName, diskName);
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/Disk/DiskRevokeAccessMethod.cs]: same questions

File: src/ResourceManager/Compute/Commands.Compute/Generated/Image/ImageDeleteMethod.cs [hunk 29]
```diff
@@ -79,8 +79,7 @@ protected void ExecuteImageDeleteMethod(object[] invokeMethodInputParameters)
             string resourceGroupName = (string)ParseParameter(invokeMethodInputParameters[0]);
             string imageName = (string)ParseParameter(invokeMethodInputParameters[1]);
 
-            var result = ImagesClient.Delete(resourceGroupName, imageName);
-            WriteObject(result);
+            ImagesClient.Delete(resourceGroupName, imageName);
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/Image/ImageDeleteMethod.cs]: breaking change?

File: src/ResourceManager/Compute/Commands.Compute/Generated/InvokeAzureComputeMethodCmdlet.cs [hunk 30]
```diff
@@ -310,6 +326,21 @@ protected override void ProcessRecord()
                     case "SnapshotUpdate":
                         ExecuteSnapshotUpdateMethod(argumentList);
                         break;
+                    case "VirtualMachineExtensionCreateOrUpdate":
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/InvokeAzureComputeMethodCmdlet.cs]: are all of these new cmdlets?

File: src/ResourceManager/Compute/Commands.Compute/Generated/Snapshot/SnapshotDeleteMethod.cs [hunk 31]
```diff
@@ -79,8 +79,7 @@ protected void ExecuteSnapshotDeleteMethod(object[] invokeMethodInputParameters)
             string resourceGroupName = (string)ParseParameter(invokeMethodInputParameters[0]);
             string snapshotName = (string)ParseParameter(invokeMethodInputParameters[1]);
 
-            var result = SnapshotsClient.Delete(resourceGroupName, snapshotName);
-            WriteObject(result);
+            SnapshotsClient.Delete(resourceGroupName, snapshotName);
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/Snapshot/SnapshotDeleteMethod.cs]: what is this code for?

File: src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachine/VirtualMachineUpdateMethod.cs [hunk 32]
```diff
@@ -0,0 +1,113 @@
+//
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachine/VirtualMachineUpdateMethod.cs]: what is this for?  Is this a new cmdlet?

File: src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineExtension/VirtualMachineExtensionCreateOrUpdateMethod.cs [hunk 33]
```diff
@@ -0,0 +1,127 @@
+//
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineExtension/VirtualMachineExtensionCreateOrUpdateMethod.cs]: same question, what is this for?

File: src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineExtension/VirtualMachineExtensionDeleteMethod.cs [hunk 34]
```diff
@@ -0,0 +1,112 @@
+//
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineExtension/VirtualMachineExtensionDeleteMethod.cs]: What is this for?

File: src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineExtension/VirtualMachineExtensionGetMethod.cs [hunk 35]
```diff
@@ -0,0 +1,136 @@
+//
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineExtension/VirtualMachineExtensionGetMethod.cs]: same question

File: src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineExtension/VirtualMachineExtensionListMethod.cs [hunk 36]
```diff
@@ -0,0 +1,113 @@
+//
+// Copyright (c) Microsoft and contributors.  All rights reserved.
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineExtension/VirtualMachineExtensionListMethod.cs]: what is this for?

File: src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineExtension/VirtualMachineExtensionUpdateMethod.cs [hunk 37]
```diff
@@ -0,0 +1,127 @@
+//
+// Copyright (c) Microsoft and contributors.  All rights reserved.
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineExtension/VirtualMachineExtensionUpdateMethod.cs]: Is this needed?

File: src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetGetOSUpgradeHistoryMethod.cs [hunk 38]
```diff
@@ -0,0 +1,99 @@
+//
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetGetOSUpgradeHistoryMethod.cs]: WHat is this for?

File: src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetGetOSUpgradeHistoryNextMethod.cs [hunk 39]
```diff
@@ -0,0 +1,85 @@
+//
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetGetOSUpgradeHistoryNextMethod.cs]: what is this for?

File: src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetReimageMethod.cs (hunk 18 above)
Comment (markcowl) [reply src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetReimageMethod.cs]: It looks liek the below actually writes the value, so, what is this method change for?

File: src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetStartMethod.cs (hunk 20 above)
Comment (markcowl) [reply src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetStartMethod.cs]: again, it looks like the method below is doign the actual work, so what is this change for?

File: src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSetRollingUpgrade/VirtualMachineScaleSetRollingUpgradeCancelMethod.cs (hunk 22 above)
Comment (markcowl) [reply src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSetRollingUpgrade/VirtualMachineScaleSetRollingUpgradeCancelMethod.cs]: Same thing -= it looks like the work for the cmdlet is done in the method below, so what is the purpose of this change?

File: src/ResourceManager/Compute/Commands.Compute/Manual/PSOperationStatusResponse.cs [hunk 40]
```diff
@@ -0,0 +1,35 @@
...
+//
+
+using System;
+using Microsoft.Azure.Management.Compute.Models;
+
+namespace Microsoft.Azure.Commands.Compute.Automation.Models
+{
+    public class PSOperationStatusResponse
+    {
+        public string Name { get; set; }
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/Manual/PSOperationStatusResponse.cs]: did this not exist before?  Are the pall the properties of the previous output type included?

**Comment by markcowl** [comment]
on demand run here: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/powershell-demand/929/

**Comment by krhynerson** [comment]
I'm also getting the null AccessSAS value on the Grant-AzureRmDiskAccess cmdlet.  

I've upgraded the AzureRM module to version 6.8.1.

What do I need to do to apply the change from @hyonholee ?

**Pull request merged**